src/source-pcap-file.c

```diff
@@ -292,13 +292,26 @@ TmEcode ReceivePcapFileThreadInit(ThreadVars *tv, const void *initdata, void **d
             CleanupPcapFileThreadVars(ptv);
             SCReturnInt(TM_ECODE_OK);
         }
+        pv->cur_dir_depth = 0;
+
+        int should_recurse = 0;
+        pv->should_recurse = false;
+        if (ConfGetBool("pcap-file.recursive", &should_recurse) == 1) {
+            pv->should_recurse = should_recurse == 1;
```

Comment:
nit: `= true`.

Also, doesn't look like `should_recurse` is used after this, so setting it is unnecessary.

Reply:
I'll clean this up.

Reply:
@victorjulien I took a closer look at this.  The function `ConfGetBool` returns the sets the variable as an int.  However, it's stored in the threadvars as a bool.  So this is being used to convert from the int value to the bool.  It is the same paradigm that's being used below with pcap-file.continuous.  Two suggestions that would be clearer are to do either:
`pv->should_recurse = (should_recurse == 1) ? true : false;`
or
`pv->should_recurse = (should_recurse == 1);`
Do you have a preference?  And should I also go ahead and update pcap-file.continuous?